### ISSUE TITLE
azfs not implemented

### ISSUE THREAD

**Issue by dineshvelmuruga** [tensorflow/io#1505]
>>> tensorflow_io.__version__
'0.20.0'
>>> tensorflow.__version__    
'2.6.0'

Code:
import os
import tensorflow as tf
import tensorflow_io. as tfio

os.environ[
    "TF_AZURE_STORAGE_KEY"
] = "<access_key>"
account_name = "<account_name"

pathname = "az://{}/".format(account_name)

data_path = pathname + "/data/taxi/data.csv"

with tf.io.gfile.GFile(data_path, mode="r") as r:
    print(r.read())

**Error:**
tensorflow.python.framework.errors_impl.UnimplementedError: File system scheme 'az' not implemented

Please let me know if I am missing something

**Comment by yongtang** [comment]
@dineshvelmuruga Which OS system are you using? Is it on Windows?

**Comment by dineshvelmuruga** [comment]
@yongtang yes. its on windows. But when I install tensorflow-io on linux issue resolved. But another issue

I am tryin out the example given in the documentation. But can not read/write to azure blob storage
Error:
InternalError: Failed to create directory az://<account_name>/test

Please not I am using access keys for TF_AZURE_STORAGE_KEY.

However, code is working with emulator.

It is really painful to fix without the error trace

**Comment by yongtang** [comment]
@dineshvelmuruga  The Windows issue is caused by https://github.com/tensorflow/tensorflow/issues/49515 , which will be resolved once PR https://github.com/tensorflow/tensorflow/pull/49520 is merged and pull into tfio.

Will take a look at the other error you encounter.
